test/common.py

```diff
@@ -2652,7 +2661,7 @@ def launch_browser_harness_with_proc_snapshot_workaround(cls, parallel_harness,
       # Give the browser time to spawn its subprocesses. Use an increasing
       # timeout as a crude way to account for system load.
       if parallel_harness or is_safari():
-        time.sleep(2 + count * 0.3)
+        time.sleep(min(2 + count * 0.3, 10))
```

Comment:
Unrelated, but this seems like a quite a lot of sleeping ... do we need to do this even on linux?

Oh wait this function is never run on linux...   maybe we could add `assert not LINUX` or update the doc string saying if/when this hack is needed?

Reply:
It is.. and maybe there is a better way, but before changing this too much, I just want to settle to getting passing runs first.